src/parsec.rs

```diff
@@ -101,11 +101,20 @@ impl<T: NetworkEvent, S: SecretId> Parsec<T, S> {
         // Return all the events we think `src` doesn't yet know about, packed into a `Response`.
         let other_parent_hash = self.peer_manager.last_event_hash(src).ok_or(Error::Logic)?;
         let other_parent = self.events.get(other_parent_hash).ok_or(Error::Logic)?;
-        let last_ancestors_hashes = other_parent
+        let mut last_ancestors_hashes = other_parent
             .last_ancestors
             .iter()
             .filter_map(|(peer_id, &index)| self.peer_manager.event_by_index(peer_id, index))
             .collect::<BTreeSet<_>>();
+        // If there are any peers for which `src` doesn't have a `last_ancestors` entry, add those
```

Comment:
I don't remember in what circumstance the src could have no last_ancestors entry...
Could you remind me (and update the comment accordingly to help the next reader)?